src/components/Clock.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import './Clock.scss';
+
+type State = {
+  currentTime: string;
+};
+
+type Props = {
+  clockName: number;
+};
+
+export class Clock extends React.Component<Props, State> {
+  timerId: NodeJS.Timer | null = null;
+
+  state: State = {
+    currentTime: new Date().toLocaleTimeString(),
+  };
+
+  componentDidMount() {
+    this.timerId = setInterval(() => {
+      this.updateCurrentTime();
+      // eslint-disable-next-line no-console
+      // console.log(this.state.currentTime);
```

Comment:
4. Change the Clock component to print the time not only on the page but also in the DevTools console.
![image](https://user-images.githubusercontent.com/37161315/143418390-287d9071-fc5c-4fbb-8ded-8de90eb219a3.png)